### PR TITLE
Delete tag (cascade delete)

### LINKED ISSUE #71: Add function that deletes a tag from all students
Add funtion which delete a tag from all students

### PR DESCRIPTION
**Pull request by ongzhili**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DeleteTagCommand.java [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+ * Deletes tags from all students that contain tags that match the given string.
+ */
+public class DeleteTagCommand extends Command {
+
+    public static final String COMMAND_WORD = "deleteTag";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Given a String, removes every tag from every student"
+            + " that matches the String\n"
+            + "Example: " + COMMAND_WORD + "Olympiad Student";
```
Comment (IamZhenHong) [inline src/main/java/seedu/address/logic/commands/DeleteTagCommand.java]: Missing a space character between COMMAND__WORD and "Olympiad Student". :)

**Review by IamZhenHong — CHANGES_REQUESTED** [review]
It seems that the deleteTag command only works for tags with capitalised first letter.

File: src/main/java/seedu/address/logic/commands/DeleteTagCommand.java (hunk 1 above)
Comment (ongzhili) [reply src/main/java/seedu/address/logic/commands/DeleteTagCommand.java]: Fixed

**Comment by ongzhili** [comment]
> It seems that the deleteTag command only works for tags with capitalised first letter.

Tags should be parsed to be auto-capitalized, so this shouldn't be an issue (there should not be any non-capitalized tags in the system)

**Review by J-wang-CSMA — APPROVED** [review]
Approved, update with simple test cases for equals function.

**Review by IamZhenHong — APPROVED** [review]

**Pull request merged**